public/landing.html

```
@@ -1,776 +1,375 @@
+                <div class="text-sm mt-8 overflow-x-auto" style="font-family: 'JetBrains Mono', 'Fira Code', 'Courier New', monospace; line-height: 1.7;">
+                    <p><span class="text-accent font-bold">import</span> <span class="text-white">Piik</span></p>
+                    <br/>
+                    <p class="text-gray-500 italic">// Generate a smart link</p>
+                    <p><span class="text-accent font-bold">let</span> <span class="text-white">link</span> <span class="text-accent">= try await</span> <span class="text-white">Piik.Links.</span><span class="text-yellow-500">create</span><span class="text-white">(</span></p>
+                    <p class="text-white">&nbsp;&nbsp;url: <span class="text-green-500">"https://your-app.com/promo"</span>,</p>
+                    <p class="text-white">&nbsp;&nbsp;domain: <span class="text-green-500">"link.brand.com"</span>,</p>
+                    <p class="text-white">&nbsp;&nbsp;tags: [<span class="text-green-500">"summer-sale"</span>, <span class="text-green-500">"social"</span>]</p>
+                    <p class="text-white"><span class="text-white">)</span></p>
+                    <br/>
+                    <p class="text-yellow-500">print<span class="text-white">(link.shortUrl)</span></p>
+                    <p class="text-gray-400 font-bold">> https://link.brand.com/x7k9Q</p>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)